### PR TITLE
ci: issue template correction

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
## What type of PR is this? (check all applicable)

- [ ] 💾 Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 🔧 Optimization
- [ ] 📝 Documentation
- [ ] ✅ Test
- [x] 🐳 Other

## Related issues 

- Related issue #https://github.com/narwhals-dev/narwhals/issues/218
- Closes #

## Checklist

- [ ] Code follows style guide (ruff)
- [ ] Tests added 
- [ ] Documented the changes

## If you have comments or can explain your changes, please do so below.

The template for the docs issues was not showing up, it had errors 😱.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/doc_issue.yml [hunk 1]
```diff
@@ -34,6 +34,4 @@ body:
     label: Would you want to open a pull request to fix it?
     options:
       - 'yes'
-      - 'no'
-      
-           
+      - 'no'
```
Comment (MarcoGorelli) [inline .github/ISSUE_TEMPLATE/doc_issue.yml:37]: just a thought on this one - do you think we risk that people think they might look bad if they answer "no"?

maybe we could rephrase, so as to not require an answer, but instead just write "if you'd be interested in opening a pull request to fix this, please let us know!"

Comment (DeaMariaLeon) [reply .github/ISSUE_TEMPLATE/doc_issue.yml]: I changed it. We have the same question on  the feature request template. I was going to change that too, but I'm not sure. Can you confirm please? If yes, on the same PR?

Comment (MarcoGorelli) [reply .github/ISSUE_TEMPLATE/doc_issue.yml]: Yeah, sure

Comment (DeaMariaLeon) [reply .github/ISSUE_TEMPLATE/doc_issue.yml]: pronto

**Review by MarcoGorelli — APPROVED** [review]
thanks @DeaMariaLeon , feel free to merge

**Pull request merged**